src/main/kotlin/lotto/model/LottoStore.kt

```diff
@@ -0,0 +1,31 @@
+package lotto.model
+
+import lotto.entity.Lotto
+import lotto.entity.LottoNumber
+import lotto.entity.LottoTicket
+import lotto.entity.Lottos
+import lotto.entity.PurchaseMoney
+
+class LottoStore(
+    private val purchaseMoney: PurchaseMoney,
+    private val manualLottoTickets: Array<LottoTicket> = arrayOf(),
+    private val lottoGenerator: LottoGenerator = RandomLottoGenerator()
+) {
+    var lottoCount = purchaseMoney.getPurchaseLottoCount()
+        private set
```

Comment:
변수에 대한 책임이 모호한거 같아요. lottoCount는 어떤것을 의미할까요?
상태값을 가질필요는 없을거같아요, 또한 외부에 노출되지 않아도 될거같아요!

```suggestion
    private val autoLottoCount get() = purchaseMoney.getPurchaseLottoCount() - manualLottoTickets.size
```

Reply:
외부와의 노출에서 고민이 되는 점이 있어서 여쭤봅니다!
Controller에서 수동, 자동 로또 구매 결과 출력을 호출할 때, 각각의 로또 개수를 출력해야해서 public으로 변수를 선언했었는데 private으로 숨기고, 이를 따로 getAuttoLottoCount 라는 함수를 구현해서 반환해주는 방식으로 해도 상관이 없을까요!? 
직접적인 변수 접근을 할 수 없어 이와 같은 방법이 좋아보이지만 필요없는 함수를 결과 출력을 위해 굳이 생성하는게 아닌가 싶어 의문이 듭니다!

```kotlin
class World {
    private fun initLottos(purchaseMoney: PurchaseMoney, manualLottoCount: LottoCount): Lottos {
        val store = LottoStore(purchaseMoney, initManualLotto(manualLottoCount))

        val totalLottos = store.makeLottos()

        outputView.printMessage(OutputView.MESSAGE_PURCHASE_COUNT, manualLottoCount.value, store.getAutoLottoCount())
        outputView.lottosResult(totalLottos)

        return totalLottos
    }
    ....
}
```


```kotlin
class LottoStore {
    private val auttoLottoCount get() = purchaseMoney.getPurchaseLottoCount() - manualLottoTickets.size

    fun getAutoLottoCount(): LottoCount = LottoCount(auttoLottoCount)
    ....
}
```

Reply:
써니님 말씀대로 불필해 보이는 함수가 되었네요
제가 커뮤니케이션 미스가 있엇네요 🙏 
상태값을 가지는 변수보단  [Backing field](https://kotlinlang.org/docs/properties.html#backing-fields)를 활용을 추천하는 코멘트였습니다!
auttoLottoCount는 getter만 있기 때문에 외부에 노출되어도 상관없을것 같아요!  🙏 🙏